tools/update_schema.ps1

```diff
@@ -33,23 +35,31 @@ if (-not $isAdmin -and -not $WhatIfPreference) {
     }
 }
 
-$destinations = @(
+$destinationsV1 = @(
```

Comment:
Don't copy paste, make a folder list and file name list (aspnetcore_schema.xml, aspnetcore_schema_v2.xml) and loop over both.